src/avro2json.c

```diff
@@ -718,6 +739,17 @@ static int avro_value_to_csv(FILE *dest, const avro_value_t *value,
     const void *val;
     size_t size;
     CHECKED_EV(avro_value_get_fixed(value, &val, &size));
+
+    if(conf->ms_hadoop_logical_types && size == 16){
```

Comment:
This means, the conversion of fixed length (16) byte array to Guid is unconditional, right?
Current logic says, if the target column is of type 'guid', then Guid recognition is applied.
Should we somehow work on a column-level here?

Reply:
Not sure what you mean by "unconditional"... Given --ms-hadoop-logical-types arg is set, we need to print as Guid every FIXED value having type=="System.Guid" (and size of 16).